recipes/sparse_dot_mkl/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "sparse_dot_mkl" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 352341c5cf739f72f49ba3df972f5fe99b08b7d2dfebe14367cfa857d0f4e9e0
+
+build:
+  skip: true  # [win]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - numpy >=1.16
+    - python
+    - scipy >=1.0
+    - mkl >=v2020.0
```

Comment:
mkl version numbers do not have a `v` in the front on conda-forge

Reply:
Resolved